packages/components/src/components/dataDisplay/Badge/Badge.stories.tsx

```diff
@@ -58,10 +77,19 @@ const AllTemplate = () => {
 
 export const All = AllTemplate.bind({});
 
-const Template = (args) => <Badge {...args}></Badge>;
+const Template = ({ showLeadingICon, showTrailingIcon, ...args }) => (
+  <Badge
+    {...(showLeadingICon && { leadingIcon: <Icon /> })}
+    {...(showTrailingIcon && { trailingIcon: <Icon /> })}
+    {...args}
+  />
+);
+
 export const Playground = Template.bind({});
 Playground.args = {
   variant: 'solid',
   status: 'primary',
   children: 'Composer badge!',
+  showLeadingIcon: true,
+  showTrailingIcon: true,
```

Comment:
I've found an issue when trying the playground arguments. I haven't been able to activate both leading and trailing icon in the story. That could be related to the icon issue I mentioned above

Reply:
I had a typo in showLeadingIcon in the Template function. Now this is fixed. I also set the option to enable both icons